keps/sig-auth/1205-bound-service-account-tokens/README.md

```diff
@@ -511,3 +586,31 @@ are properly reloading tokens by:
 - **Will enabling / using this feature result in non-negligible increase of
   resource usage (CPU, RAM, disk, IO, ...) in any components?** it adds a
   token minting operation in the API server every ~48 minutes for every pod.
+
+### Troubleshooting
+
+The Troubleshooting section currently serves the `Playbook` role. We may consider
+splitting it into a dedicated `Playbook` document (potentially with some monitoring
+details). For now, we leave it here.
+
+_This section must be completed when targeting beta graduation to a release._
+
+- **How does this feature react if the API server and/or etcd is unavailable?**
+
+- **What are other known failure modes?**
```

Comment:
* failure to issue token via token subresource (appears in apiserver log? apiserver metrics?)
* failure to create root CA config map (appears in kube-controller-manager log? k-c-m metrics?)
* kubelet fails to renew token (appears in kubelet log? in kubelet metrics?)
* workload fails to refresh token from disk (shows up in apiserver `serviceaccount_stale_tokens_total` metric)

Reply:
done